sklearn/mixture/gmm.py

```diff
@@ -112,7 +114,7 @@ def sample_gaussian(mean, covar, covariance_type='diag', n_samples=1,
     return (rand.T + mean).T
 
 
-class GMM(BaseEstimator):
+class _GMMBase(_MixtureBase):
```

Comment:
This should not inherit from the new `_MixtureBase` class. Just `BaseEstimator` as previously. We do not want to change the implementation and behavior of the old `GMM` class in any way besides issuing a deprecation warning in the init.